### PR TITLE
Add command args to container creation

### PR DESCRIPTION
**Pull request by CodingParsley**
### Description
Add args in addition to command for container creation
Sample use: https://github.com/confluentinc/cc-druid/pull/509
This change is backward compatible

### CODE REVIEW

File: controllers/druid/testdata/broker-statefulset.yaml [hunk 1]
```diff
@@ -32,8 +32,10 @@ spec:
       affinity: {}
       containers:
       - command:
-        - bin/run-druid.sh
-        - broker
+        - sh
+        args:
+        - "-c for i in /opt/druid/conf/druid/cluster/env/*.env; do source $i; done && /druid.sh"
```
Comment (harinirajendran) [inline controllers/druid/testdata/broker-statefulset.yaml]: shouldn't it be bin/druid.sh?

File: controllers/druid/testdata/broker-deployment.yaml [hunk 2]
```diff
@@ -35,8 +35,10 @@ spec:
       affinity: {}
       containers:
       - command:
-        - bin/run-druid.sh
-        - broker
+        - sh
+        args:
+        - "-c for i in /opt/druid/conf/druid/cluster/env/*.env; do source $i; done && /druid.sh"
```
Comment (harinirajendran) [inline controllers/druid/testdata/broker-deployment.yaml]: shouldn't it be bin/druid.sh?

File: controllers/druid/testdata/broker-statefulset.yaml (hunk 1 above)
Comment (CodingParsley) [reply controllers/druid/testdata/broker-statefulset.yaml]: i think it's `/druid.sh`, according to our current statefulsets

File: apis/druid/v1alpha1/druid_types.go [hunk 3]
```diff
@@ -50,9 +50,15 @@ type DruidSpec struct {
...
 	DeleteOrphanPvc bool `json:"deleteOrphanPvc"`
 
-	// Required: path to druid start script to be run on container start
+	// Required: Command to be run on container start
 	StartScript string `json:"startScript"`
 
+	// Optional: bash/sh entry arg, eg. `sh -c ${EntryArg}`
+	EntryArg string `json:"entryArg,omitempty"`
+
+	// Optional: druid shell script path
```
Comment (harinirajendran) [inline apis/druid/v1alpha1/druid_types.go]: lets also add an example here. `eg. sh -c ${EntryArg} ${DruidScript} nodeType`

**Review by harinirajendran — APPROVED** [review]

**Pull request merged**